tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -1081,6 +1081,39 @@ describe(__filename, () => {
     expect(root).toHaveState('pictureData', null);
   });
 
+  it('fetches the user notifications after having deleted a profile picture', () => {
+    const username = 'tofumatt';
+    const { store } = dispatchSignInActions({
+      userProps: {
+        ...defaultUserProps,
+        picture_url: 'https://example.org/pp.png',
+        username,
+      },
+    });
+    const dispatchSpy = sinon.spy(store, 'dispatch');
+    const errorHandler = createStubErrorHandler();
+
+    const root = renderUserProfileEdit({ errorHandler, store });
+    const user = getCurrentUser(store.getState().users);
+
+    dispatchSpy.reset();
+    sinon.assert.notCalled(dispatchSpy);
```

Comment:
This seems a bit odd to assert that the `dispatchSpy` was not called immediately after you `reset` it. Doesn't resetting it guarantee that it won't have been called at this point?